python/grizzly/Makefile

```diff
@@ -2,8 +2,13 @@ OS=$(shell uname -s)
 LLVM_VERSION=$(shell llvm-config --version | cut -d . -f 1,2)
 
 PYTHON_HEADER_INCLUDE = $(shell python-config --includes)
-NUMPY_HEADER_INCLUDE = -I$(shell python -c "import numpy; print numpy.get_include()")
+NUMPY_HEADER_INCLUDE = -I$(shell python -c "import numpy; print numpy.get_include()" 2>/dev/null)
 PYTHON_LDFLAGS = $(shell python-config --ldflags)
+
+ifeq (${NUMPY_HEADER_INCLUDE}, -I)
+  $(error NumPy is needed to be installed)
```

Comment:
Thanks for the contribution! can you change the message to:

"Error: NumPy installation not found"

Reply:
O.K. I'll modify it.